src/Modules/CalcPerform.lua

```diff
@@ -2133,49 +1947,59 @@ function calcs.perform(env, avoidCache)
 					level = value.level,
 					quality = 0,
 				})
-				local curseModList = { }
+
+				local modScale = nil
+				if value.applyToPlayer then
+					local cfg = { skillName = grantedEffect.name }
+					local inc = modDB:Sum("INC", cfg, "CurseEffectOnSelf") + gemModList:Sum("INC", nil, "CurseEffectAgainstPlayer")
+					local more = modDB:More(cfg, "CurseEffectOnSelf") * gemModList:More(nil, "CurseEffectAgainstPlayer")
+					local configInc = value.effect and (1 + value.effect / 100) or 1
+					modScale = (1 + inc / 100) * more * configInc
```

Comment:
```suggestion
					modScale = (1 + inc / 100) * more * configInc
```
Does map curse effect act as more multiplier or does it stack additively with regular increased curse effect on player.

Reply:
The effect box has been removed so that value is not longer needed.